src/shared/utils/fileSystem.ts

```diff
@@ -0,0 +1,23 @@
+import { BYTE_QUANTITY } from 'shared/types/fileSystem';
+
+type FORMATTER = (value?: string | number) => string;
+export const byteFormatter: FORMATTER = (cellValue) => {
+  if (!cellValue && cellValue !== 0) return '';
+
+  let output = cellValue;
+  [BYTE_QUANTITY.b, BYTE_QUANTITY.kb, BYTE_QUANTITY.mb, BYTE_QUANTITY.gb, BYTE_QUANTITY.tb].some(
```

Comment:
Why some() method? It might be confusing using this method not directly he was made for.